src/nomad_analysis/jupyter/schema.py

```diff
@@ -65,14 +65,14 @@
 
 if TYPE_CHECKING:
     from nomad.datamodel.datamodel import (
-        EntryArchive,
         ArchiveSection,
+        EntryArchive,
     )
     from structlog.stdlib import (
         BoundLogger,
     )
 
-m_package = Package(name='analysis_jupyter')
+m_package = Package()
```

Comment:
You need to import schemapackage instead of package

```suggestion
m_package = SchemaPackage()
```

Reply:
Import it at line 58, I cannot add it from this review

Reply:
I see, will change it